tests/functional/api/test_statistics.py

```diff
@@ -0,0 +1,35 @@
+"""
+GitLab API: https://docs.gitlab.com/ee/api/statistics.html
+"""
+
+from tests.functional.conftest import reset_gitlab
+
+
+def test_get_statistics(gl):
+
+    # delete resources created from other functional tests to reset application statistics
+    reset_gitlab(gl)
+
+    gl.users.create(
+        {
+            "email": "veryuniqueemail@test.com",
+            "username": "plsdontclash",
+            "name": "uniqueuser",
+            "password": "hopefullythisdoesnthavetobeunique",
+        }
+    )
+    gl.groups.create({"name": "gryffindor", "path": "gryffindor"})
+    gl.groups.create({"name": "slytherin", "path": "slytherin"})
+    gl.snippets.create(
+        {
+            "title": "uniquesnippet",
+            "file_name": "snippet394.py",
+            "content": "import gitlab",
+        }
+    )
+
+    statistics = gl.statistics.get()
+    assert statistics.snippets.isdigit() == True
+    assert statistics.users.isdigit() == True
+    assert statistics.groups.isdigit() == True
+    assert statistics.projects.isdigit() == True
```

Comment:
To make flake8 happy:

```suggestion
    assert statistics.snippets.isdigit()
    assert statistics.users.isdigit()
    assert statistics.groups.isdigit()
    assert statistics.projects.isdigit()
```